src/main/kotlin/cz/loono/backend/Application.kt

```diff
@@ -21,14 +22,23 @@ fun main(args: Array<String>) {
 }
 
 @Configuration
-class Config(private val authenticator: BearerTokenAuthenticator) : WebMvcConfigurer {
+class Config(
+    private val authenticator: BearerTokenAuthenticator,
+    private val onboardRequiredInterceptor: OnboardRequiredInterceptor
+) : WebMvcConfigurer {
     val unauthenticatedEndpoints = listOf(
         "/v3/api-docs",
         "/error",
     )
 
     override fun addInterceptors(registry: InterceptorRegistry) {
-        registry.addInterceptor(authenticator).excludePathPatterns(unauthenticatedEndpoints)
+        registry.addInterceptor(authenticator)
+            .excludePathPatterns(unauthenticatedEndpoints)
+            .order(0)
+
+        registry.addInterceptor(onboardRequiredInterceptor)
```

Comment:
Nezahrnuje novy interceptor take samotny /onboard endpoint? Je vubec mozne napred naonboardovat?

Reply:
Jo, vypada to tak. Zrovna jsem si dneska rikal, ze potrebujeme vic testovat tu integraci.